src/components/tasks/card/index.tsx

```diff
@@ -33,18 +31,30 @@ const Card: FC<Props> = ({ content }) => {
   const localEndsOn = new Date(parseInt(endsOn, 10) * 1000);
   const fromNowEndsOn = moment(localEndsOn).fromNow();
 
-  const statusFontColor = status === ACTIVE || ASSIGNED || COMPLETED || PENDING ? '#00a337' : '#f83535';
+  const statusFontColor = status === ACTIVE || ASSIGNED || COMPLETED || PENDING
+    ? '#00a337'
+    : '#f83535';
 
   const iconHeight = '25px';
   const iconWidth = '25px';
 
+  const cardClassNames = [classNames.card];
+  const currentDate = new Date();
+  if (status !== COMPLETED) {
+    if (+localEndsOn - (+currentDate) <= 0) {
```

Comment:
Is there any special reason to use `+` before the variables?

Reply:
I'll test it without the + symbol and let you know 😁

Reply:
refactored. Please check